cli/src/template/init/mopro-wasm/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "mopro-wasm"
```

Comment:
Is it possible that we can just reuse the `mopro-wasm` crate in mopro
e.g.
```toml
[dependencies]
mopro-wasm = { git = "https://github.com/zkmopro/mopro" }
```
(maybe we also add a feature flag like `mopro-ffi` to have a `halo2` feature flag)

Ideally we want to use the same crate in the root
(can it select different toolchain for different targets?
I think it would be also fine if we use nightly toolchain for `mopro-ffi` buildings?)

Reply:
That's exactly what I want to achieve, but I haven't been able to make it work. I need to understand more about how `wasm-pack` works before attempting to write a lower-level build script.
In the meantime, it might be worth trying your suggestion of using the nightly toolchain for mopro-ffi and the CLI.